### DISCUSSION TITLE
Migrating hardware/updating to beta

### DISCUSSION THREAD

**Discussion by bubba925** [blakeblackshear/frigate#8754] [Ask A Question]
Curious on what the recommended process is for this. I got curious and decided to try and reduce power usage in my Unraid server. I could get Frigate running on a separate box that runs proxmox and only had HA on it. I have spun up a separate ubuntu VM and have frigate up and running on the latest beta. I am letting it run for a bit without any recording to ensure I do not run into any strange issues before decommissioning the unraid instance. iGPU pass through and coral passthrough seem to be working great, still using the network unraid share for storage of the recordings. 

The question is, is there a good way to transition to using this new instance that is running the beta? If I enable recording with it pointed at the current location of recordings will it pick up those recordings? Do I need to just wipe all the old recordings and start new?

Appreciate the help and everything from this project.

I have tried searching for something along these lines and can not seem to find anything, maybe an FAQ could be added for migrating in the docs?

**Comment by NickM-27** [comment]
you would need to keep the storage location the same and migrate the database

**Reply by bubba925** [reply]
well that makes sense. The beta puts the database default in the config folder now correct?

**Reply by NickM-27** [reply]
yes
